src/utils/network-monitor.ts

```diff
@@ -955,17 +892,18 @@ export class NetworkMonitor {
     }
   }
 
-  blockJobHandler = async (network: string, job?: BlockJob): Promise<void> => {
-    if (job !== undefined) {
+  async blockJobHandler(network: string, job?: BlockJob): Promise<void> {
+    if (job) {
       this.latestBlockHeight[job.network] = job.block
+
       if (this.verbose) {
```

Comment:
@alexanderattar i think you can use your new verbose structurelogger here.

Reply:
Yeah good catch